vm/src/obj/objbyteinner.rs

```diff
@@ -1184,14 +1191,14 @@ impl PyByteInner {
                 new_value.extend(&self.elements);
             }
 
-            Ok(new_value)
+            new_value
         }
     }
 
-    pub fn irepeat(&mut self, n: isize, _vm: &VirtualMachine) -> PyResult<()> {
+    pub(crate) fn irepeat(&mut self, n: isize) {
         if self.elements.is_empty() {
             // We can multiple an empty vector by any integer, even if it doesn't fit in an isize.
-            return Ok(());
+            return;
```

Comment:
I think this type is useful for other crates that might use RustPython -- why do you think this should be `pub(crate)`?

Reply:
Thanks for pointint it out. I was not thinking of rustpython crate users at all. Let me check `pub(crate)`s again

Reply:
I reverted most of them except for a few cases that private is obvious